frontend/components/navigation/Tabs/CardsTab.tsx

```diff
@@ -4,30 +4,32 @@ import { ChevronDown, ChevronRight } from "react-feather";
 import { Card as CardType } from "../../../types/Card";
 import { CardList } from "../../flashcard/CardList";
 import { Flashcard } from "../../flashcard/Flashcard";
+import { Deck } from "@/types/Deck";
```

Comment:
```suggestion
import { Deck as DeckType} from "@/types/Deck";
```
just for clarification